frontend/src/bundles/video-editor/components/menu-body/menu-body.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+    Box,
+    CloseButton,
+    Flex,
+    Heading,
+    VStack,
+} from '~/bundles/common/components/components.js';
+
+type Properties = {
+    title: string | React.ReactNode;
+    children: React.ReactNode;
+    isOpen: boolean;
+    onClose: () => void;
+};
+
+const MenuBody: React.FC<Properties> = ({
+    title,
+    children,
+    isOpen,
+    onClose,
+}) => {
+    return (
+        <>
+            {isOpen && (
+                <Box
+                    sx={{
+                        position: 'fixed',
+                        top: '50%',
+                        transform: 'translateY(-50%)',
+                        left: '100px',
+                        height: '72vh',
+                        width: '335px',
+                        p: 2,
+                        zIndex: 1000,
+                        borderRadius: '8px',
+                        boxShadow: 'lg',
+                        bg: 'background.900',
+                        color: 'white',
+                    }}
+                >
+                    <Flex justifyContent="space-between" alignItems="center">
+                        <Heading as="h3" size="md">
+                            {title}
+                        </Heading>
```

Comment:
You should add variant to the heading. We have already merged business styles into next.

Reply:
@o-nedashkivska Thanks, i am already using the variant h3, after reviewing the figma dsesing i remove the size property as we dont need it.